message_sender/factory.py

```diff
@@ -287,11 +287,16 @@ class WhatsAppApiSenderException(Exception):
 
 
 class WhatsAppApiSender(object):
-    def __init__(self, api_url, token, hsm_namespace, hsm_element_name, session=None):
+    def __init__(
+        self, api_url, token, hsm_namespace, hsm_element_name, ttl, session=None
+    ):
         self.api_url = api_url
         self.token = token
         self.hsm_namespace = hsm_namespace
         self.hsm_element_name = hsm_element_name
+        self.ttl = ttl
+        self.time_period = 0
+        self.max_time_period = 30
```

Comment:
I don't think we're using the `time_period` and `max_time_period` variables anymore.

Reply:
thanks for pointing this out